src/main/kotlin/racing/model/RacingHistory.kt

```diff
@@ -1,5 +1,28 @@
 package racing.model
 
-data class RacingHistory(
-    val result: ArrayDeque<Car>
-)
+class RacingHistory(
+    val roundHistories: ArrayDeque<RoundHistory>,
+) {
+
+    constructor() : this(ArrayDeque())
```

Comment:
```suggestion
class RacingHistory(
    val roundHistories: ArrayDeque<RoundHistory> = ArrayDeque()
) {
```
이런식으로 작성해도 괜찮을 것 같네요 ㅎ

Reply:
오 훨씬 깔끔해졌습니다. 제가 코틀린을 좀더 공부해야겠다고 느끼게 됩니다!